cmd/depository/main.go

```diff
@@ -106,43 +138,19 @@ func run() error {
 	hf := depository.Group("hf")
 	hf.Get("metadata", hfHandler.GetMetadata)
 
-	// basic handlers
-	basicContract, err := contracts.NewBasic(client, *contract)
-	if err != nil {
-		return err
-	}
-	basicHandler := handler.NewBasicHandler(basicContract)
+	basicHandler := handler.NewBasicHandler(basicContract, dbHandler)
 	// basic routes
 	basic := depository.Group("basic")
 	basic.Get("currentNonce", basicHandler.CurrentNonce)
 	basic.Get("total", basicHandler.Total)
 	basic.Post("putValue", basicHandler.PutValue)
 	basic.Get("getValue", basicHandler.GetValue)
 	basic.Post("verifyValue", basicHandler.VerifyValue)
+	basic.Get("depositories", basicHandler.List)
```

Comment:
how about remove app group `depository`  as this could be duplicated in route path?